test/functional/sync_plan/info_test.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../test_helper'
+require 'hammer_cli_katello/sync_plan'
+require_relative 'sync_plan_helpers'
+require_relative '../organization/organization_helpers'
+module HammerCLIKatello
+  describe SyncPlan::InfoCommand do
+    include OrganizationHelpers
+    include SyncPlanHelpers
+
+    it 'requires ID' do
```

Comment:
This test is testing that the InfoCommand "allows ID". That'd be a better name. The use of "requires <option>" generally tests that the command fails without the option specified, like [this](https://github.com/Katello/hammer-cli-katello/blob/master/test/functional/repository/update_test.rb#L18).